Sources/Target/Windows/Thread.cpp

```diff
@@ -119,11 +124,24 @@ void Thread::updateState(DEBUG_EVENT const &de) {
     _state = kStopped;
 
     DS2LOG(
-        Debug, "exception from inferior, tid=%lu, code=%s, address=%" PRI_PTR,
+        Debug, "exception from inferior, tid=%lu, code=%s, dwFirstChance=%lu, address=%" PRI_PTR,
         tid(),
         Stringify::ExceptionCode(de.u.Exception.ExceptionRecord.ExceptionCode),
+        de.u.Exception.dwFirstChance,
         PRI_PTR_CAST(de.u.Exception.ExceptionRecord.ExceptionAddress));
 
+      // First-chance exceptions may yet be caught by the debugee, so don't break for them.
+      // An exception is STATUS_BREAKPOINT, since Process::initialize()'s implementation relies on
```

Comment:
I'm afraid I don't understand this comment.  We do not control the `DebugEvent` right?  So, the `ExceptionCode` being `STATUS_BREAKPOINT` is defined by the system is my understanding.

Reply:
Yes that's my understanding as well. Not sure where the mismatch with the comment is in your eyes.

Reply:
Was able to narrow the condition and simplify the comment.